perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2451,7 +2430,9 @@ public Member setSponsoredMember(PerunSession session, Vo vo, User userToBeSpons
 			validateMember(session, sponsoredMember);
 		}
 
-		getPerunBl().getUsersManagerBl().validatePassword(session, userToBeSponsored, namespace);
+		if (data.getNamespace() != null && !data.getNamespace().isBlank()) {
```

Comment:
You can use library method `StringUtils.isNotBlank()`, it should handle also null input. This can be fixed on multiple places.

Reply:
Thanks for the tip. I have used this method and also the `isBlank()` as it makes the code much more readable.